homeassistant/components/broadlink/heartbeat.py

```diff
@@ -0,0 +1,55 @@
+"""Heartbeats for Broadlink devices."""
+import datetime as dt
+import logging
+
+import broadlink as blk
+
+from homeassistant.const import CONF_HOST
+from homeassistant.helpers import event
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class BroadlinkHeartbeat:
+    """Manages heartbeats in the Broadlink integration.
+
+    Some devices reboot when they cannot reach the cloud. This mechanism
+    feeds their watchdog timers so they can be used offline.
+    """
+
+    INTERVAL = dt.timedelta(minutes=2)
```

Comment:
Please move this to const.py and rename to e.g. HEARTBEAT_INERVAL

Reply:
I think that renaming it to HEARTBEAT_INTERVAL is a good idea, the code gets more explicit. But I don't think it's interesting to move the constant out of this scope, as it only makes sense within this class.

Reply:
Accepted.